bin/rialto/runtime/src/millau_messages.rs

```diff
@@ -214,7 +214,7 @@ impl TargetHeaderChain<ToMillauMessagePayload, bp_millau::AccountId> for Millau
 	fn verify_messages_delivery_proof(
 		proof: Self::MessagesDeliveryProof,
 	) -> Result<(LaneId, InboundLaneData<bp_rialto::AccountId>), Self::Error> {
-		messages::source::verify_messages_delivery_proof::<WithMillauMessageBridge, Runtime>(proof)
+		messages::source::verify_messages_delivery_proof::<WithMillauMessageBridge, Runtime, ()>(proof)
```

Comment:
```suggestion
		messages::source::verify_messages_delivery_proof::<WithMillauMessageBridge, Runtime, crate::MillauGrandpaInstance>(proof)
```

This will make it more clear to the reader.

You'll need a typedef for this though, just like how it's done [here](https://github.com/paritytech/parity-bridges-common/blob/master/bin/millau/runtime/src/lib.rs#L319).

Reply:
done it